pyanaconda/ui/gui/spokes/advanced_storage.py

```diff
@@ -668,7 +668,7 @@ def on_clear_icon_clicked(self, entry, icon_pos, event):
         if icon_pos == Gtk.EntryIconPosition.SECONDARY:
             entry.set_text("")
 
-    def on_page_switched(self, notebook, new_page, new_page_num, *args):
+    def on_page_switched(self, notebook, _new_page, new_page_num, *args):
```

Comment:
I have a bad feeling about mixing the `del` and this approach. I mean it's hard to distinguished when to use which pattern. Also if the situation will change and this method will be inherited and used by other class, you have to rename this.... I'm honestly not sure if that is correct or wrong :thinking:

Reply:
I would really recommend sticking to one of those & not to combine both. Frankly, the `_` seems like the less confusing one & I've seen it used in the past, unlike the "del" method, which also potentially could result in some serious confusion later, eq. when you leave a stray "del" somewhere in the function/method & then the data you feed to the function gets mysteriously wiped. :P